src/pages/About/About.tsx

```diff
@@ -0,0 +1,42 @@
+import oskmLogo from '../../resource/oskm-logo.svg';
+import AboutDetail from './AboutDetail';
+import { useToggle } from '../../hooks/useToggle';
+
+const About = () => {
+    const [isOn, toggle] = useToggle();
+
```

Comment:
could you put the content inside of a map nad put the type in the type.ts to simulate the real content